invenio_records_resources/resources/records/resource.py

```diff
@@ -28,6 +30,15 @@ def __init__(self, config=None, service=None):
         super(RecordResource, self).__init__(config=self.load_config(config))
         self.service = service or RecordService()
 
+    def _get_es_preference(self):
+        user_agent = resource_requestctx.headers.get('User-Agent', "")
+        # TODO: Find a way to pass request.remote_addr in resource_requestctx
+        ip = request.remote_addr
```

Comment:
+1 to the TODO 💯 . Otherwise, I don't see the point of having the `resource_requestctx` if we also access `request`.

Reply:
Thanks for reminding me. Made issue here: https://github.com/inveniosoftware/flask-resources/issues/85